automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/lab/LabNotebookPage.scala

```diff
@@ -0,0 +1,146 @@
+package org.broadinstitute.dsde.workbench.leonardo.lab
+
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.commons.text.StringEscapeUtils
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.scalatest.Matchers.convertToAnyShouldWrapper
+import org.scalatest.concurrent.Eventually
+import org.scalatest.concurrent.PatienceConfiguration.{Interval, Timeout}
+import org.scalatest.exceptions.TestFailedDueToTimeoutException
+import org.scalatest.time.{Seconds, Span}
+import org.broadinstitute.dsde.workbench.leonardo.KernelNotReadyException
+import org.openqa.selenium.{By, TimeoutException, WebDriver, WebElement}
+
+import scala.collection.JavaConverters._
+import scala.concurrent.duration._
+import scala.language.postfixOps
+import scala.util.{Failure, Success, Try}
+
+
+class LabNotebookPage(override val url: String)(override implicit val authToken: AuthToken, override implicit val webDriver: WebDriver)
+  extends LabPage with NotebookPanel with Eventually with LazyLogging {
+
+
+  override def open(implicit webDriver: WebDriver): LabNotebookPage = {
+    super.open.asInstanceOf[LabNotebookPage]
+  }
+
+  // menu elements
+
+  lazy val fileMenu: Element = {
```

Comment:
The menu items should be moved to `Toolbar` trait

Reply:
The traits don't have a webDriver, so I can't move items that find specific cssSelectors to them. The traits only contain the strings that serve as cssSelectors